src/layout/Cell.js

```diff
@@ -41,7 +41,7 @@
         var context = this;
         var offsetMultiple;
 
-        var dragHandles = element.selectAll(".dragHandle").data(this._dragHandles, function (d) { return d; });
+        var dragHandles = element.selectAll("#"+this.id()+" > .dragHandle").data(this._dragHandles, function (d) { return d; });
```

Comment:
No it would be as is, just to replace```this._element.selectAll``` or ```this.element().selectAll``` or ```element.selectAll``` with a ```this.selectAll```